integration_tests/iceberg-sink/README.md

```diff
@@ -34,3 +34,45 @@ docker compose exec presto presto-cli --server localhost:8080
 ```sql
 select user_id, count(*) from iceberg.demo_db.demo_table group by user_id
 ```
+
+## Demo: Iceberg Compaction With Airflow
+
+We can use airflow to trigger iceberg's compaction task at regular intervals, It can avoid the slowdown of queries due to too many small files.
+
+1. Add directory and build docker image
```

Comment:
Is it possible to do all the following settings automatically by doing a simple `docker compose up`?